Parsers/ASimAuditEvent/Parsers/ASimAuditEvent.yaml

```diff
@@ -33,6 +33,7 @@ Parsers:
   - _ASim_AuditEvent_SentinelOne
```

Comment:
Can you please include illumio details in EventVendor and EventProduct fields for the schemas you wrote parser for i.e. Audit, Network and Authentication?
https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv

Reply:
![image](https://github.com/user-attachments/assets/ad207367-9cd4-4804-811a-789efbb90f8e)

Reply:
![image](https://github.com/user-attachments/assets/1c6e2345-413e-4a2d-a5d0-1dc1331a571c)

Reply:
![image](https://github.com/user-attachments/assets/b9de6b29-24e7-438c-a6af-b2383fe02a8d)